snabble/Android-SDK
# Custom checkout header

### What's new?

The header for the checkout with the sco can now be customized by implementing the snabble_checkout_header in the given project. 

### How to test?

* set up snabble_checkout_header  inside the sample app (custom layout choice)
* run the following steps -> checkin -> give camera permissions -> buy something with payment method "EC-Terminal"
* if the snabble_checkout_header is set up it should be seen  above the qr code
* else the old checkout  with the helper text should be seen
